mkdocs/docs/HPC/program_examples.md

```diff
@@ -1,6 +1,9 @@
 {% set exampledir="examples/Program-examples" %}
 # Program examples { #ch:program-examples}
 
+If you have **not done already** copy our examples to your home directory:
```

Comment:
```suggestion
If you have **not done so already** copy our examples to your home directory by running the following command:
```

Maybe also add a note to explain what `~` is (or point to a section where that's explained)